include/irr/asset/IAssetManager.h

```diff
@@ -182,6 +182,19 @@ namespace asset
 			Furthermore if an Asset Bundle hasn't been loaded to cache before, it becomes loaded into cache memory and the function returns it. Otherwise
 			it is simply returned.
 
+			Take a look on @param _hierarchyLevel
+			Remember that even if you are looking for a Texture Asset, it doesn't mean it is on specified 
+			certain const level relatively! So you can't assume it is always const x level.
+			Sometimes you may be intresed only in texture Asset, so you can make it \broot\b.
+
+			For more details about hierarchy levels.
+			@see IAssetLoader
+
+			There is a term in reference to above - \bDowngrade\b. 
+			You can find Downgrades in Assets definitions. For instance ICPUMesh::IMAGEVIEW_HIERARCHYLEVELS_BELOW.
+			The syntax is simple - \b(Current level + Downgrade)\b. You ought to pass just like that a expression to _hierarchyLevel.
```

Comment:
clarify that this function is meant to be used by IAssetLoaders when loading dependent assets.